src/app/games/quarto/QuartoTutorial.ts

```diff
@@ -16,59 +20,64 @@ const BBAA: QuartoPiece = QuartoPiece.BBAA;
             new QuartoState([
-                [BBBA, AABB, ABBA, EMPTY],
-                [EMPTY, EMPTY, EMPTY, EMPTY],
-                [EMPTY, BAAA, EMPTY, EMPTY],
-                [EMPTY, EMPTY, EMPTY, EMPTY],
-            ], 7, QuartoPiece.ABAA),
-            new QuartoMove(2, 2, QuartoPiece.BAAB),
+                [BBBA, AABB, ABBA, ____],
+                [____, ____, ____, ____],
+                [____, BAAA, ____, ____],
+                [____, ____, ____, ____],
+            ], 7, ABAA),
+            new QuartoMove(2, 2, BAAB),
             $localize`Perfect!`,
         ),
         TutorialStep.fromMove(
             $localize`Situation`,
-            $localize`We have here a tricky situation.<br/><br/>
-        Analyze the board and play your move, carefully paying attention not to let the opponent win on the next move.`,
+            $localize`We have here a tricky situation.<br/><br/>Analyze the board and play your move, carefully paying attention not to let the opponent win on the next move.`,
             new QuartoState([
-                [BBBB, BBBA, BBAB, EMPTY],
-                [ABAA, BABB, BBAA, EMPTY],
-                [ABAB, BAAA, BAAB, EMPTY],
-                [EMPTY, EMPTY, EMPTY, EMPTY],
-            ], 7, QuartoPiece.AABA),
-            [new QuartoMove(3, 3, QuartoPiece.AABB)],
+                [BBBB, BBBA, BBAB, ____],
+                [ABAA, BABB, BBAA, ____],
+                [ABAB, BAAA, BAAB, ____],
+                [____, ____, ____, ____],
+            ], 7, AABA),
+            [new QuartoMove(3, 3, AABB)],
             $localize`Well done!`,
             $localize`Failed! Your opponent can either align:<ul><li>the squares horizontally on the third line or vertically on the third column,</li><li>small pieces horizontally on the first line or vertically on the second or third column,</li><li>empty pieces horizontally on the first line or vertically on the first column,</li><li>clear pieces diagonally.</li></ul>`,
         ),
+        TutorialStep.fromPredicate(
+            TutorialStepMessage.ADVANCED_RULES(),
+            $localize`Quarto has advanced setup to make games more complicated for one, or both players.<br/>When you create a game and you want to change the original configuration, you can add level to one player. When your level increases, it becomes easier for you to win, so if first player is weaker at this game, make first player level two and the game will be more even and interresting.<br/>The level two rules correspond to added 2x2 squared.<br/>In this board, Light can win by making a square.<br/><br/>You're playing Light, win.`,
```

Comment:
Faut trouver ici le pattern qu'on va utiliser pour ces règles. On parle de "règles avancées", "règles configurables", "options de configurations" ? J'aurais bien mis "A game of Quarto can be configured to make it more complicated for one or both players. [...]" Et comme titre, "Configurable rules", ou "Configuration: player level" (plus uniforme pour les jeux où on a plusieurs options de config). Qu'en dites vous ?

Reply:
À propager dans le FR too!

Reply:
j'ai-ce-tu rien oubliette ?

Reply:
t'as oublié pas mal oui :D

Reply:
what ?

Reply:
une grosse partie de ma review dans ce message a été ignorée ! Le texte est toujours tout croche !